src/element-templates/cmd/RemoveElementTemplateHandler.js

```diff
@@ -133,6 +137,33 @@ export default class RemoveElementTemplateHandler {
 
     return newBo;
   }
+
+  /**
+   * Copy documentation and execution listeners from source to target business object.
+   *
+   * NOTE: Mutetes the target business object.
+   * @param {ModdleElement} sourceBusinessObject
+   * @param {ModdleElement} targetBusinessObject
+   */
+  _copyNonTemplateProperties(sourceBusinessObject, targetBusinessObject) {
+    const copy = this._moddleCopy;
+    const bpmnFactory = this._bpmnFactory;
+
+    const { documentation, extensionElements } = sourceBusinessObject;
+
+    if (documentation) {
+      const docs = copy.copyProperty(documentation, targetBusinessObject, 'documentation');
+      targetBusinessObject.documentation = docs;
+    }
+
+    if (extensionElements && extensionElements.values) {
+      const exts = copy.copyProperty(extensionElements, targetBusinessObject, 'extensionElements');
```

Comment:
[Q]: We copy all extension elements here, why don't we copy only the individual values that we want to keep? 

[Q2]: Does it make sense to statically configure what we want to keep (in a variable)?

Reply:
Haven't found a way to copy execution listeners with `ModdleCopy#copyProperty` directly.

Moreover, with `isAny` checking for either C7 or C8 type, there is one implementation that works for both and doesn't need to be aware of engine version.

Is there a smarter way to copy only execution listeners?